frontend/components/CategorySelectionOnFnd.tsx

```
@@ -0,0 +1,150 @@
+              <option key={category.id} value={category.id}>
+                {category.name}
+              </option>
+            ))}
+          </select>
+        </div>
+
+        {selectedCategory && (
+          <div className="mb-6">
+            <label className="block text-gray-700 font-medium mb-2">
+              Skills: <span className="text-sm text-gray-500">(Select up to 5)</span>
+            </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)